migrations/20230329194448_Add_chatgpt_results_table.sql

```diff
@@ -0,0 +1,12 @@
+create table statement_predictions (
```

Comment:
What is the primary key for this table?
Do we need specific indices for lookups?